charts/fybrik/files/taxonomy/taxonomy.json

```diff
@@ -70,6 +73,43 @@
     "ProcessingLocation": {
       "type": "string"
     },
+    "Property": {
```

Comment:
Please put in an infrastructure taxonomy.  Also, property is a pretty generic name.  It's not too clear what it refers to.